docs/manage/CLI.md

```diff
@@ -188,6 +188,51 @@ viam organizations --help
 
 ## Commands
 
+### board
+
+The `board` command allows you to manage your board definition files.
+With it, you can upload new board definition files to the Viam app, download your existing board definition files, and list the files already uploaded.
+
+Board definition files are used to configure pin mappings for [`customlinux` boards](/components/board/customlinux/).
+
+```sh {class="command-line" data-prompt="$"}
+viam board upload --name=<board name> --organization=<org name> --version=<definition file version> file.json
+viam board download --name=<board name> --organization=<org name> --version=<definition file version>
+viam board list --organization=<org name>
+```
```

Comment:
I think those are the same as below?
```suggestion
```

Reply:
The CLI page approach is:

```
# Usage with <placeholders>
```

Examples:

```
# Usage with examples
```


Similar to how we do JSON Template and JSON Example tabs for components. You can see this down in `module` or up in `data`.

Too similar to be of use though? I can remove if you like.

Reply:
Gonna ship with this per elsewhere on page, but agreed that we can certainly revisit this approach!